history/taskana-simplehistory-provider/src/test/java/acceptance/events/task/CreateHistoryEventOnTaskTransferAccTest.java

```diff
@@ -70,40 +82,48 @@ Stream<DynamicTest> should_CreateTransferredHistoryEvent_When_TaskIsTransferred(
           assertThat(events).hasSize(1);
           assertTransferHistoryEvent(
               events.get(0).getId(),
-              "WBI:100000000000000000000000000000000001",
+              q.getThird(),
               "WBI:100000000000000000000000000000000006",
               "admin");
         };
 
-    return DynamicTest.stream(testCases.iterator(), Triplet::getLeft, test);
+    return DynamicTest.stream(testCases.iterator(), Quadruple::getFirst, test);
   }
 
   @WithAccessId(user = "admin")
   @TestFactory
   Stream<DynamicTest> should_CreateTransferredHistoryEvents_When_TaskBulkTransfer() {
-    List<Triplet<String, List<String>, Consumer<List<String>>>> testCases =
+    List<Triplet<String, Map<String, String>, Consumer<List<String>>>> testCases =
         List.of(
             Triplet.of(
                 "Using WorkbasketId",
-                List.of(
+                Map.of(
                     "TKI:000000000000000000000000000000000021",
-                    "TKI:000000000000000000000000000000000022"),
+                    "WBI:100000000000000000000000000000000001",
+                    "TKI:000000000000000000000000000000000022",
+                    "WBI:100000000000000000000000000000000001",
+                    "TKI:000000000000000000000000000000000002",
+                    "WBI:100000000000000000000000000000000006"),
                 wrap(
                     (List<String> taskIds) ->
                         taskService.transferTasks(
                             "WBI:100000000000000000000000000000000007", taskIds))),
             Triplet.of(
                 "Using WorkbasketKey and Domain",
-                List.of(
+                Map.of(
                     "TKI:000000000000000000000000000000000023",
-                    "TKI:000000000000000000000000000000000024"),
+                        "WBI:100000000000000000000000000000000001",
+                    "TKI:000000000000000000000000000000000024",
+                        "WBI:100000000000000000000000000000000001",
+                    "TKI:000000000000000000000000000000000055",
+                        "WBI:100000000000000000000000000000000015"),
```

Comment:
I prefer Map.ofEntries() for a better readability. What do you think?

Reply:
Ok :)